### ISSUE TITLE
Spammy log: failed to get energy consumption

### ISSUE THREAD

**Issue by nieksand** [NVIDIA/tensorrt-inference-server#101]
I'm testing TRTIS on an AWS g3s instance which has an Nvidia M60 inside.

Every two seconds the logs emit:
```
E0220 07:59:01.944339 1 metrics.cc:238] failed to get energy consumption for GPU 0, NVML_ERROR 3
E0220 07:59:03.955762 1 metrics.cc:238] failed to get energy consumption for GPU 0, NVML_ERROR 3
E0220 07:59:05.963320 1 metrics.cc:238] failed to get energy consumption for GPU 0, NVML_ERROR 3
E0220 07:59:07.974456 1 metrics.cc:238] failed to get energy consumption for GPU 0, NVML_ERROR 3
```

I assume this GPU just doesn't support that metric.  

Perhaps warn just once or never rather than repeating constantly?

(I did not have this issue when using a p3 instance type which has a V100).

**Comment by deadeyegoodwin** [comment]
This was fixed a couple of week ago on master. Metric will be attempted 3 times and then no more attempts will be made, so at most you will see 3 messages. https://github.com/NVIDIA/tensorrt-inference-server/commit/0b41a2f9e2532a605b520b1aa7bebe4169c9041d#diff-9391bb025a876f53948323cd6abbaffa

Fix will be in 19.03 release.

**Comment by nieksand** [comment]
Cool.  Thanks!
